From f726b368da87024a0cfe82627e0730ffe4578716 Mon Sep 17 00:00:00 2001
From: Marvin Schmidt <marv@exherbo.org>
Date: Wed, 18 Sep 2019 17:07:29 +0200
Subject: [PATCH] build: lower version requirement for EGL

`egl.pc` can be provided either by mesa or libglvnd. The latter doesn't
follow the same version scheme as mesa but instead uses the API version
that the library exposes, which is 1.5 for EGL[1]

[1] https://github.com/NVIDIA/libglvnd/commit/0dfaea2bcb7cdcc785f95e244223bd004a2d7fba#diff-b58a140c00ea99fb9a708e15afaade62R8
---
 wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript b/wscript
index 3569dbf060..97a4d9196b 100644
--- a/wscript
+++ b/wscript
@@ -636,7 +636,7 @@ video_output_features = [
         'deps': 'wayland',
         'groups': [ 'gl' ],
         'func': check_pkg_config('wayland-egl', '>= 9.0.0',
-                                 'egl',         '>= 9.0.0')
+                                 'egl',         '>= 1.5')
     } , {
         'name': '--gl-win32',
         'desc': 'OpenGL Win32 Backend',
